WebRoot/js/zimbraMail/calendar/controller/ZmCalViewController.js

```diff
@@ -2181,6 +2191,33 @@ function(appt, mode) {
 	}
 };
 
+/**
+ * Check if deleting item(s) is opened
+ *
+ * @param {Array}   toDelete   list of items to delete
+ */
+ZmCalViewController.prototype.isDeletingItemOpen =
+function(toDelete) {
```

Comment:
Can we move the implementation of  `ZmCalViewController.prototype.isDeletingItemOpen` and `ZmBriefcaseController.prototype.isDeletingItemOpen` to the base class 'ZmController' ?

Reply:
We can move them to a base class like ZmBaseController. However, `if - else if` statement needs to be added to handle controller-specific or view-specific process.
I think it is better to handle such specific process(es) in each child controller.